moh-hni-bathl7/src/main/java/com/cgi/bathl7/BatHL7Processor.java

```diff
@@ -31,9 +31,6 @@
 public class BatHL7Processor {
 
 	private static final String LINE_SEPARATOR = "---------------------------------------";
-
```

Comment:
Add line between constant declarations to be consistent with other classes code.